common/src/commonTest/kotlin/com/github/mustafaozhan/ccc/common/repo/SettingsRepositoryTest.kt

```diff
@@ -79,12 +80,88 @@ class SettingsRepositoryTest {
     }
 
     @Test
-    fun lastReviewRequest() {
+    fun default_lastReviewRequest() {
         given(settings)
             .function(settings::getLong)
             .whenInvokedWith(matching { it == KEY_LAST_REVIEW_REQUEST }, any())
             .thenReturn(DEFAULT_LAST_REVIEW_REQUEST)
 
         assertEquals(DEFAULT_LAST_REVIEW_REQUEST, repository.lastReviewRequest)
     }
+
+    // setters
+    @Test
+    fun set_firstRun() {
+        given(settings)
+            .function(settings::putBoolean)
+            .whenInvokedWith(matching { it == KEY_FIRST_RUN }, any())
+            .thenReturn(Unit)
+
+        repository.firstRun = true
+
+        verify(settings)
+            .function(settings::putBoolean)
```

Comment:
How come you're using the callable reference API over the invocation API, and not verifying the value being set as well?

```suggestion
            .invocation { putBoolean(KEY_FIRST_RUN, true) }
            .wasInvoked()
```

Reply:
Yes after applying your previous suggestions it makes more sense, previously I was supplying `any` and verifying `any` but now after I update the default stubs it makes much more sense, thank you!